include/net/nrf_cloud_coap.h

```diff
@@ -135,6 +135,18 @@ int nrf_cloud_coap_pgps_url_get(struct nrf_cloud_rest_pgps_request const *const
  */
 int nrf_cloud_coap_sensor_send(const char *app_id, double value, int64_t ts_ms);
 
+/**
+ * @brief Send raw bytes to nRF Cloud.
+ * @param[in]     app_id The app_id identifying the type of data.
```

Comment:
*question:* why does this need an appId, after all raw implies that the cloud shouldn't care about it and app ID is a nRF Cloud specific feature that allows it to distinguish messages for showing them in the UI.

Reply:
I would definitely prefer a more direct channel for custom messages. If we can support it in the backend, we should.

Reply:
This kind of architecture change needs to wait until all the parties involved are back from vacation.

Reply:
But should we keep some kind of categorization way. The AppID provides info how to handle the message, what is the time-to-leave, who has subscribed it etc. For now the AppID is for UI but I see many other needs for it

I would understand that devices could send encrypted messages but still categorize those.